org.mixedrealitytoolkit.input/Interactors/Poke/PokeInteractor.cs

```diff
@@ -49,11 +59,27 @@ protected virtual bool TryGetPokePose(out Pose pose)
         protected virtual bool TryGetPokeRadius(out float radius)
         {
             HandJointPose jointPose = default;
-            if (xrController is ArticulatedHandController handController
-                && (XRSubsystemHelpers.HandsAggregator?.TryGetNearInteractionPoint(handController.HandNode, out jointPose) ?? false))
+
+#pragma warning disable CS0618 // Type or member is obsolete
+#pragma warning disable CS0612 // Type or member is obsolete
+            if (forceDeprecatedInput)
```

Comment:
style....collapse these `if` statements into no `if`

Reply:
Oops, I missed that, thx for catching that.  Fixed in https://github.com/MixedRealityToolkit/MixedRealityToolkit-Unity/pull/753/commits/2689162e4b7fed5ad17bec76fa484ebb1c420953 , resolving.